red-model/src/main/java/com/masi/red/entity/CandidateAnswer.java

```diff
@@ -0,0 +1,46 @@
+package com.masi.red.entity;
+import com.masi.red.common.Language;
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Data
+@Entity
+@Table(name = "candidate_answer")
+public class CandidateAnswer {
+
+    @Id
+    @GeneratedValue(generator = "optimized-sequence")
+    @Column(name = "id", nullable = false, unique = true)
+    private int id;
+
+    @NotNull
+    @ManyToOne(cascade = {CascadeType.ALL})
```

Comment:
Wydaje mi się, że ALL to za dużo, bo nie chcemy żeby przy ewentualnym usunięciu odpowiedzi kandydata usunął nam się kandydat.